operational/stranded_servers/stranded_servers.pt

```diff
@@ -142,46 +142,62 @@ script "js_get_stranded_instance", type: "javascript" do
   EOS
 end
 
-###############################################################################
-# Policy
-###############################################################################
-
-policy "pol_stranded_instances" do
-  validate $ds_get_stranded_instances do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Stranded Instance Details"
-    detail_template <<-EOS
-# Stranded Instance Details
-| Cloud Name | Cloud Type | Instance Name | Resource UID | Updated On | Is Instance Locked | Href | Tags |
-| ---------- | ---------- | ------------- | ------------ | ---------- | ------------------ | ---- | ---- |
-{{ range data -}}
-| {{.cloud_name}} | {{.cloud_type}} | {{.name}} | {{.resource_uid}} | {{.updated_at}} | {{.locked}} | {{.href}} | {{.tags}} |
-{{ end -}}
-    EOS
-    escalate $report_stranded_instances
-    escalate $approve_terminate_instances
-    check eq(size(data),0)
-  end
-end
-
 ###############################################################################
 # Escalations
 ###############################################################################
 
-escalation "report_stranded_instances" do
+escalation "esc_email" do
+  automatic true
+  label "Send Mail"
+  description "Sends incident email"
   email $param_email
 end
 
-escalation "approve_terminate_instances" do
-  request_approval  do
+escalation "esc_approve_terminate_instances" do
+  automatic false
   label "Approve Instance Termination"
-    description "Approve escalation to run RightScale Cloud Workflow to delete Stranded Instnace"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "[Note]: Locked instances will not be terminated."
+  description "Approve escalation to run RightScale Cloud Workflow to delete Stranded Instnace"
```

Comment:
Change to "Delete selected stranded instances"

Reply:
"Delete selected stranded instances" will be more than 20 characters so I have changed it to "Terminate Instances"